index.bs

```diff
@@ -649,6 +649,7 @@ When this method is invoked, the user agent MUST execute the following algorithm
         1. Otherwise, [=list/Append=] |C| to |excludeCredentialDescriptorList|.
     1. [=In parallel=], invoke the [=authenticatorMakeCredential=] operation on |authenticator| with |rpId|,
         |clientDataHash|, |options|.{{MakeCredentialOptions/rp}}, |options|.{{MakeCredentialOptions/user}},
+        <code>|options|.|authenticatorSelection|.{{AuthenticatorSelectionCriteria/requireResidentKey}}</code>,
```

Comment:
I think you can replace `|authenticatorSelection|` with `{{MakeCredentialOptions/authenticatorSelection}}` in order to remove it from `Ignored Vars`, but IM me if that gives you bikeshed errors.